### PR TITLE
Highlight video

### PR DESCRIPTION
**Pull request by youralien**
Summary:
- Refactor of video.py to be more reusable
- Addition of functions to create highlight video
- new test_video.py file that has one method test_create_highlight_video

Tests:
- With the refactor, we can still run the normal pipeline and get a tracked video out from TrafficGUI
- With the new code, we can produce highlight videos programmatically

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,4 +8,5 @@
     else:
         project = sys.argv[1]
         project_identifier = project.strip('/').split('/')[-1]
-        api.runTrajectoryAnalysis(project_identifier)
\ No newline at end of file
+        api.runTrajectoryAnalysis(project_identifier)
+        # api.createVideos(project_identifier)
```
Comment (segerphilip) [inline main.py:12]: Is this a carry over, or just planned implementation?

File: trafficcloud/video.py [hunk 2]
```diff
@@ -149,3 +135,124 @@ def get_resolution(videopath):
...
+    subprocess.call(["ffmpeg", 
+        "-framerate", get_framerate(video_path), 
+        "-i", os.path.join(images_folder, images_prefix+"%d.png"),
+        "-filter:v", "setpts={:0.1f}*PTS".format(pts_multiplier),
+        "-c:v", "libx264", 
+        "-pix_fmt", "yuv420p", 
+        os.path.join(videos_folder, filename)])
+
+def create_title_slide(width, height, save_path, object_id1, object_id2, fontsize=None, textcolor='#FFFFFF', facecolor='#000000'):
+    """
```
Comment (segerphilip) [inline trafficcloud/video.py:226]: Move this docstring above the function?

**Review by segerphilip — APPROVED** [review]
Looks good, test and 🚢

File: trafficcloud/video.py (hunk 2 above)
Comment (youralien) [reply trafficcloud/video.py:226]: Above might be good for when we're doing the official API docs, but most Python functions have docstrings below function header based on my experience.

File: main.py (hunk 1 above)
Comment (youralien) [reply main.py:12]: @runnersaw you were editing this file.  Have any thoughts what happened here?

**Pull request merged**